Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/PolDiffILLReduction.py

```diff
@@ -0,0 +1,591 @@
+
+        self.declareProperty(WorkspaceGroupProperty('AbsorberInputWorkspace', '',
+                                                    direction=Direction.Input,
+                                                    optional=PropertyMode.Optional),
+                             doc='The name of the absorber workspace group.')
+
+        self.setPropertySettings('AbsorberInputWorkspace',
+                                 EnabledWhenProperty(quartz,
+                                                     EnabledWhenProperty(vanadium, sample, LogicOperator.Or),
+                                                     LogicOperator.Or))
+
+        self.declareProperty(MatrixWorkspaceProperty('BeamInputWorkspace', '',
+                                                     direction=Direction.Input,
+                                                     optional=PropertyMode.Optional),
+                             doc='The name of the empty beam input workspace.')
+
+        self.setPropertySettings('BeamInputWorkspace', transmission)
+
+        self.declareProperty(MatrixWorkspaceProperty('AbsorberTransmissionInputWorkspace', '',
+                                                     direction=Direction.Input,
+                                                     optional=PropertyMode.Optional),
+                             doc='The name of the absorber transmission input workspace.')
+
+        self.setPropertySettings('AbsorberTransmissionInputWorkspace', EnabledWhenProperty(transmission, beam, LogicOperator.Or))
+
+        self.declareProperty(MatrixWorkspaceProperty('TransmissionInputWorkspace', '',
+                                                     direction=Direction.Input,
+                                                     optional=PropertyMode.Optional),
+                             doc='The name of the transmission input workspace.')
+
+        self.setPropertySettings('TransmissionInputWorkspace', reduction)
+
+        self.declareProperty(WorkspaceGroupProperty('ContainerInputWorkspace', '',
+                                                    direction=Direction.Input,
+                                                    optional=PropertyMode.Optional),
+                             doc='The name of the container workspace.')
+
+        self.setPropertySettings('ContainerInputWorkspace', reduction)
+
+        self.declareProperty(WorkspaceGroupProperty('QuartzInputWorkspace', '',
+                                                    direction=Direction.Input,
+                                                    optional=PropertyMode.Optional),
+                             doc='The name of the polarisation efficiency correction workspace.')
+
+        self.setPropertySettings('QuartzInputWorkspace',
+                                 EnabledWhenProperty(vanadium, sample, LogicOperator.Or))
+
+        self.declareProperty(name="OutputTreatment",
+                             defaultValue="OutputIndividualScans",
```

Comment:
This is confusing, individual measurement is not a scan, the scan is the ensemble of the measurements, right?
If so change this to `Individual` or `Average`